### DISCUSSION TITLE
Github OAuth에 대해

### DISCUSSION THREAD

**Discussion by kjklalala2** [heading1/WYLSBingsu#218] [Polls]
저희가 기획에서 간편 로그인을 위해 OAuth로 Github를 선택하였는데 작업을 진행하던 도중 예상치 못한 문제점이 발생하였습니다.

바로 인증을 하고 데이터를 받아오는 것인데요   
 
![image](https://user-images.githubusercontent.com/93240443/178094886-4fa7e350-863a-483e-8c4f-10a105242aa0.png)   

다음과 같이 이메일과 name 등등이 null로 설정 되어있습니다.     

이와 같은 이유는 다음과 같습니다.   
![image](https://user-images.githubusercontent.com/93240443/178094920-2110ba13-11a2-4efa-ae77-d0040e999bf5.png)

바로 Github로그인 한 사용자가 다음과 같은 설정을 하지 않았기 때문입니다. 

그래서 코드에서는
 
![image](https://user-images.githubusercontent.com/93240443/178094977-085dbeed-b0f0-4780-b365-2f1e7d66927d.png)

에러를 통해 사용자가 각자 Github홈페이지에서 설정해주는 방법밖에 없는 것 같습니다. 

이러한 설정을 안한 유저가 많다고 생각이 들어서 저희가 기획했던 간편 로그인과 거리가 멀어지는 것 같습니다. 

따라서 이에 따른 의견을 받을려고 합니다.

**Comment by jinux127** [comment]
kakao oauth 로 투표를 하긴 했지만 지금의 로그인만으로도 좋을 것 같습니다.

**Comment by kjklalala2** [comment]
투표에 따라 빠른 시일 내에 Kakao Oauth 추가할게요!
